### ISSUE TITLE
Does not work well with `gitflow`.

### ISSUE THREAD

**Issue by JounQin** [zdharma/fast-syntax-highlighting#147]
Correct with `gc -m`:
![image](https://user-images.githubusercontent.com/8336744/61191872-e5840e00-a6e1-11e9-90cd-a1316eae405c.png)

Incorrect with `gc -am`:
![image](https://user-images.githubusercontent.com/8336744/61191886-fd5b9200-a6e1-11e9-8a96-039c1d4029e1.png)

**Comment by psprint** [comment]
Yes this is a known issue, you can use -a -m for now, I'm going to fix this till tomorrow.

**Comment by psprint** [comment]
I've added a temporary fix, it should work now. I'll fix the main issue on days

**Comment by psprint** [comment]
This is a nice candidate for Hacktoberfest 2019. The pull-request should introduce a loop in the line 179 of the file `chroma/main-chroma.ch`:

```zsh
if [[ -z "${FAST_HIGHLIGHT[chroma-${FAST_HIGHLIGHT[chroma-current]}-option-with-arg-active]}" ]]; then 
    chroma/main-chroma-print -rl -- "-z OPT-WITH-ARG-ACTIVE == true"
    if [[ "$__wrd" = -* ]]; then 
    # Here a loop if $__wrd is a single-hyphen, multi-letter string
```

Basically, FSH should split `-str` into `-s`, `-t`, and `-r` and then look for a match in the chroma definition. It currently expects only `-s`, etc. at input.

**Comment by psprint** [comment]
It would be cool if a small donation, like $5 or $10, would be granted for this.
